src/evo/netinfo.cpp

```diff
@@ -147,6 +147,12 @@ std::string NetInfoEntry::ToStringAddrPort() const
         m_data);
 }
 
+std::shared_ptr<NetInfoInterface> NetInfoInterface::MakeNetInfo(const uint16_t nVersion)
+{
+    assert(nVersion > 0 && nVersion < /*ProTxVersion::ExtAddr=*/3);
```

Comment:
Please use the enum; even if it results in a circular depends, as this is logically currently a circular dependency

Reply:
Resolved in latest pushed, updated `test/lint/lint-circular-dependencies.py` to reflect this.